installer/sakura-common.iss

```diff
@@ -48,70 +48,164 @@ MinVersion=6.1
 [Languages]
 Name: "ja"; MessagesFile: "compiler:Languages\Japanese.isl"
 Name: "en"; MessagesFile: "compiler:Default.isl"; InfoBeforeFile: "instmaterials\info_us.txt"
+Name: "zh_hans"; MessagesFile: "compiler:Languages\ChineseSimplified.isl"; InfoBeforeFile: "instmaterials\info_zh_hans.txt"
```

Comment:
https://github.com/sakura-editor/sakura/pull/1469/files#r538538926 suggested below.

```suggestion
Name: "zh_hans"; MessagesFile: "Languages\ChineseSimplified.isl"; InfoBeforeFile: "instmaterials\info_zh_hans.txt"
```

this should also be applied to non-simplified version.